SQuAD/train/dawn_qanet_1tpu.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "Google",
+    "authorEmail": "cloud-tpu-dawnbench@googlegroups.com",
+    "framework": "TensorFlow v1.8",
+    "model": "QANet",
```

Comment:
Is code available for this submission? We allow submissions without code, but whenever possible we encourage people to make their submissions reproducible.

Reply:
Great! If that is the case, I think it would be best to merge this request as is and for you to submit a separate pull request to update these files once the code is released.

Please remove the `costPerHour` field in the meantime, and I'll merge this in.